portfolio/src/main/java/com/google/sps/data/BlogHashMap.java

```diff
@@ -24,7 +24,7 @@
 import java.util.Set;
 
 public final class BlogHashMap {
-  private final String[] INTERNAL_TAGS = {"#general", "#wellbeing", "#music", "#games"};
+  private final String[] INTERNAL_TAGS = {"#general", "#wellbeing", "#music", "#games", "Tags"};
```

Comment:
Why is there tag name "Tags" which looks inconsistent with other tags?

Reply:
That was just there as a default tag since my branch didn't have the dropdown menu

Reply:
It seems for that purpose, it would still be good to have something like "#default_tags" for consistency. But your call.